src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: admin
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

You referenced to the PK userId, so you need to add 
```suggestion
                    nullable: false
                    foreignKeyName: fk_users_roles_users
                    referencedTableName: users
                    referencedColumnNames:  id
```